src/Hl7.Fhir.Core/Model/Source/search-parameters.xml

```diff
@@ -13159,7 +13159,8 @@
         <xpath value="f:Composition/f:encounter | f:DeviceRequest/f:encounter | f:DiagnosticReport/f:encounter | f:DocumentReference/f:context/f:encounter | f:Flag/f:encounter | f:List/f:encounter | f:NutritionOrder/f:encounter | f:Observation/f:encounter | f:Procedure/f:encounter | f:RiskAssessment/f:encounter | f:ServiceRequest/f:encounter | f:VisionPrescription/f:encounter" />
         <xpathUsage value="normal" />
         <target value="Encounter" />
-        <target value="EpisodeOfCare" />
+        <!-- removed, see: https://github.com/FirelyTeam/fhir-net-api/issues/1298 -->
```

Comment:
I agree that we should change the source to solve this issue. But should we not also change this file in `Hl7.Fhir.Specification.data`?

Reply:
Turns out this specific file only exists in Source (unlike profiles-types and the others).

Reply:
I just removed it

Reply:
Thanks @mmsmits. So `Hl7.Fhir.Specification.data.search-parameter.xml` did exist. But has now been synchronized with `Hl7.Fhir.Core/Model/Source/search-parameters.xml`